is.js

```diff
@@ -176,6 +176,10 @@
         return toString.call(value) === '[object RegExp]';
     };
 
+    is.map = function(value) {
+        return toString.call(value) === '[object Map]';
+    }
+
```

Comment:
This will run into problems in IE11 and earlier Safari's where the result is `'[object Object]'`.
The check then becomes coercing it's constructor to a string and seeing 'Map' in it or something.